### PR TITLE
Listener to send message expiration message

### PR DESCRIPTION
**Pull request by vitormattos**
PS: Branch created from #7594. Necessary to review and merge first the parent branch.

### CODE REVIEW

File: lib/Service/RoomService.php [hunk 1]
```diff
@@ -547,29 +549,6 @@ public function setMessageExpiration(Room $room, Participant $participant, int $
 			->where($update->expr()->eq('id', $update->createNamedParameter($room->getId(), IQueryBuilder::PARAM_INT)));
 		$update->executeStatement();
 
-		$room->setMessageExpiration($seconds);
```
Comment (nickvergessen) [inline lib/Service/RoomService.php:550]: Should be kept

File: lib/Service/RoomService.php [hunk 2]
```diff
@@ -537,8 +537,10 @@ public function verifyPassword(Room $room, string $password): array {
 		];
 	}
 
-	public function setMessageExpiration(Room $room, Participant $participant, int $seconds): void {
-		$event = new ModifyRoomEvent($room, 'messageExpiration', $seconds, null, $participant);
+	public function setMessageExpiration(Room $room, int $seconds): void {
+		$oldValue = $room->getMessageExpiration();
+		$room->setMessageExpiration($seconds);
```
Comment (nickvergessen) [inline lib/Service/RoomService.php:542]: Should be done later so it's the same as in all other methods (and the before event still has the old value in the room object

File: tests/integration/features/bootstrap/FeatureContext.php [hunk 3]
```diff
@@ -2588,6 +2588,20 @@ public function waitForXSeconds($seconds): void {
 		sleep($seconds);
 	}
 
+	/**
+	 * @When apply message expiration job manually
+	 */
+	public function applyMessageExpirationJob(): void {
```
Comment (nickvergessen) [inline tests/integration/features/bootstrap/FeatureContext.php:2594]: needs a rebase

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: lib/Service/RoomService.php (hunk 1 above)
Comment (vitormattos) [reply lib/Service/RoomService.php:550]: kept when rebased from master

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**